js/main.js

```diff
@@ -22,5 +57,68 @@ function getCoordsFloat(min, max, number) {
   }
 }
 
-getCoordsInt(1, 5);
-getCoordsFloat(1.2, 1.3, 3);
+const getType = () => TYPES[getCoordsInt(0, TYPES.length - 1)];
+
+const getTime = () => TIMES[getCoordsInt(0, TIMES.length - 1)];
+
+function getFeatures() {
+  const featuresNumber = getCoordsInt(0, FEATURES.length - 1);
+  const copy = FEATURES.slice();
+  let feature = '';
+  const checkedFeatures = [];
+  for(let i = 0; i < featuresNumber; i++) {
+    feature = copy.splice(getCoordsInt(0, copy.length - 1), 1);
+    checkedFeatures.push(...feature);
+  }
+  return checkedFeatures;
+}
+
+const getDescr = () => DESCRIPTIONS[getCoordsInt(0, DESCRIPTIONS.length - 1)];
+
+function getPhotos() {
+  const photosNumber = getCoordsInt(MINPHOTOSNUMBER, MAXPHOTOSNUMBER);
+  const photosList = [];
+  for(let i = 0; i < photosNumber; i++) {
+    photosList.push(PHOTOS[getCoordsInt(0, PHOTOS.length - 1)]);
+  }
+  return photosList;
+}
+
+function getObj(i) {
+  const locationLat = getCoordsFloat(MINLAT, MAXLAT, FIXTONUMBER);
+  const locationLng = getCoordsFloat(MINLNG, MAXLNG, FIXTONUMBER);
+  const obj = {
+    author: {
+      avatar: `img/avatars/user${(i < 10) ? `0${i}` : i}.png`,
+    },
+    offer: {
+      title: 'Дом моей мечты',
+      address: `${locationLat}, ${locationLng}`,
+      price: getCoordsInt(MINPRICE, MAXPRICE),
+      type: getType(),
+      rooms: getCoordsInt(MINROOMS, MAXROOMS),
+      guests: getCoordsInt(MINGUESTS, MAXGUESTS),
+      checkin: getTime(),
+      checkout: getTime(),
+      features: getFeatures(),
+      description: getDescr(),
+      photos: getPhotos(),
+    },
+    location: {
+      lat: locationLat,
+      lng: locationLng,
+    },
+  };
+  return obj;
+}
+
+const arr = () => {
+  const array = [];
+  for(let i = 0; i < 10; i++) {
+    array.push(getObj(i+1));
```

Comment:
Можно сделать так, тогда не нужно будет прибавлять единицу

```suggestion
  for(let i = 1; i <= 10; i++) {
    array.push(getObj(i));
  }
```